mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -211,10 +211,8 @@ class CompletionProvider(
               // The ideal solution would be to not use the `($0)` snippet when the commit character `(` is used,
               // however language servers can't distinguish what commit character is used. It works as
               // expected in IntelliJ.
-            } else if (r.sym.isError) {
-              () // No commit character
             } else {
-              item.setCommitCharacters(List(".").asJava)
+              () // No commit character
```

Comment:
@olafurpg do we want to replace it or just altogether remove?

Reply:
altogether remove the commit character pattern match

Reply:
Alright, removed it.